configuration/core-cc-runner-itools-config.yml

```
@@ -74,4 +57,31 @@ load-flow-default-parameters:
+
+search-tree-second-preventive-rao:
+  execution-condition: COST_INCREASE
+  re-optimize-curative-range-actions: false
+  hint-from-first-preventive-rao: false
+
+search-tree-load-flow-and-sensitivity-computation:
+  load-flow-provider: OpenLoadFlow
+  sensitivity-provider: OpenLoadFlow
+
+search-tree-multi-threading:
+  available-cpus: 14
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)